estate/models/estate_property.py

```diff
@@ -0,0 +1,34 @@
+from odoo import fields, models
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "estate property"
+    name = fields.Char('Property Name', required=True)
+    description = fields.Text('Description', required=False)
```

Comment:
By defaut, `string` is the variable name (without the trailing `_id(s)`), so no need to define it here. Same for `required` which is False by default